### PR TITLE
Create CHANGELOG.md :tada:

### PR DESCRIPTION
**Pull request by ismail-syed**
* Created a CHANGELOG.md file because they are important 👍 

giphy

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+## [Unreleased]
+
+## [15.1.0] - 2017-01-17
```
Comment (GoodForOneFare) [inline CHANGELOG.md]: `15.1.0` is already released, so let's make this `15.1.1`.

**Review by lemonmade — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+### Added
+- Added `eslint-index` package ([#4](https://github.com/Shopify/eslint-plugin-shopify/pull/4))
+- Added `rules-status` and `rules-omitted` scripts ([#4](https://github.com/Shopify/eslint-plugin-shopify/pull/4))
+- Added new `eslint-plugin-react` rules: `no-array-index-key`, `prefer-stateless-function`, `require-default-props` ([#4](https://github.com/Shopify/eslint-plugin-shopify/pull/4))
```
Comment (GoodForOneFare) [inline CHANGELOG.md]: `prefer-stateless-function` should be in the `Changed` section.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- Added new `eslint-plugin-lodash` rules: `import-scope` ([#4](https://github.com/Shopify/eslint-plugin-shopify/pull/4))
+- Added new `eslint-plugin-promise` rules: `no-return-wrap`, `no-nesting`, `no-promise-in-callback`, `no-callback-in-promise`, `avoid-new` ([#4](https://github.com/Shopify/eslint-plugin-shopify/pull/4))
+
```
Comment (GoodForOneFare) [inline CHANGELOG.md:12]: Missing `promise/prefer-await-to-then` and `promise/prefer-await-to-callbacks`.

**Review by GoodForOneFare — APPROVED** [review]
Nice one.  Minor stuff to address.

**Comment by GoodForOneFare** [comment]
🐑

**Pull request merged**